app/src/main/java/io/pslab/activity/LuxMeterActivity.java

```diff
@@ -239,18 +272,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 startActivity(MAP);
                 break;
             case R.id.settings:
-                Intent settingIntent = new Intent(this, SettingsActivity.class);
-                settingIntent.putExtra("title", "Lux Meter Settings");
-                startActivity(settingIntent);
+                startActivity(new Intent(this, SettingsActivity.class));
```

Comment:
This one redirects to Lux Meter settings. May be your branch is not updated. Try update it and it will fix all these.